packages/driver/src/datatypes/datetime.ts

```diff
@@ -233,10 +233,10 @@ export class LocalDate {
 
   toString(): string {
     const year =
-      this.year < 1000 || this.year > 9999
+      this.year < 0 || this.year > 9999
         ? (this.year < 0 ? "-" : "+") +
           Math.abs(this.year).toString().padStart(6, "0")
-        : this.year.toString();
+        : this.year.toString().padStart(4, "0");
```

Comment:
This was failing, but it doesn't seem related to any changes I made here. I'm not sure why this is now breaking, but perhaps it was always flaky in tests since it is randomly generated? At any rate, here is the same bit of code from the polyfill we use:

```ts
  if (year < 0 || year > 9999) {
    const sign = year < 0 ? '-' : '+';
    const yearNumber = MathAbs(year);
    yearString = sign + `000000${yearNumber}`.slice(-6);
  } else {
    yearString = `0000${year}`.slice(-4);
  }
```
https://github.com/js-temporal/temporal-polyfill/blob/a5eda13f7ce9d2f2df899c2ac56e92936ec8d1bf/lib/ecmascript.ts#L2679-L2685

I think this update matches that logic precisely.

Reply:
Looks like the polyfill version got updated from 0.4.2 to 0.4.3 in the lockfile, and the new version contains this change. Eventually when the `Temporal` api is finalised and makes its way into browsers/node/etc. we'll be able to throw all these custom datetime types away 🎉.

Reply:
Ahh, of course I should've checked the `yarn.lock` file not just the `package.json` 🤦 .

> Eventually when the Temporal api is finalised and makes its way into browsers/node/etc. we'll be able to throw all these custom datetime types away 🎉.

Yeah, what has been our policy for runtime support in the past? If something is so easily polyfilled like this, it feels like it's fine to put that on the consumer to polyfill, yeah? I guess the same question could be said for our `fetch` polyfill, so maybe we can just make some kind of executive decision about it.